<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="load_Btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Załaduj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showData_Btn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>410</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pokaż dane</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_Btn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>410</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyczyść</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>571</width>
      <height>331</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>256</width>
      <height>192</height>
     </size>
    </property>
    <property name="rowCount">
     <number>20</number>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>231</width>
      <height>401</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Wybierz kolumny:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="col3_checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Kolumna 3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="col2_checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>93</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Kolumna 2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="col4_checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>133</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Kolumna 4</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="col1_checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>73</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Kolumna 1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="apply_Btn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Zastosuj</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Wybierz typ wykresu:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="barChart_Btn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Pokaż wykres słupkowy</string>
     </property>
    </widget>
    <widget class="QPushButton" name="circleChart_Btn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Pokaż wykres kołowy</string>
     </property>
    </widget>
    <widget class="QPushButton" name="lineChart_Btn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Pokaż wykres liniowy</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>884</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionDrukuj"/>
    <addaction name="actionEksplorator_plik_w"/>
    <addaction name="actionZako_cz"/>
   </widget>
   <widget class="QMenu" name="menuOpcje">
    <property name="title">
     <string>Opcje</string>
    </property>
    <widget class="QMenu" name="menuPoka_wykres">
     <property name="title">
      <string>Pokaż wykres</string>
     </property>
     <addaction name="actionWykres_s_upkowy_2"/>
     <addaction name="actionWykres_liniowy"/>
     <addaction name="actionWykres_ko_owy"/>
    </widget>
    <addaction name="actionZa_aduj"/>
    <addaction name="separator"/>
    <addaction name="actionPoka_dane"/>
    <addaction name="actionWyczy"/>
    <addaction name="menuPoka_wykres"/>
    <addaction name="actionDodaj_opis"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionO_programie"/>
    <addaction name="actionQt"/>
    <addaction name="actionGitHub"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Pomoc</string>
    </property>
    <addaction name="actionZg_o_problem"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuOpcje"/>
   <addaction name="menuInfo"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionZa_aduj"/>
   <addaction name="actionDrukuj"/>
   <addaction name="actionEksplorator_plik_w"/>
   <addaction name="actionZako_cz"/>
   <addaction name="separator"/>
   <addaction name="actionDodaj_opis"/>
   <addaction name="actionWyczy"/>
   <addaction name="actionWykres_s_upkowy_2"/>
   <addaction name="actionWykres_liniowy"/>
   <addaction name="actionWykres_ko_owy"/>
   <addaction name="separator"/>
   <addaction name="actionZg_o_problem"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionZako_cz">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/close.ico</normaloff>:/icons/icons/close.ico</iconset>
   </property>
   <property name="text">
    <string>Zakończ</string>
   </property>
  </action>
  <action name="actionEksplorator_plik_w">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/explorer.ico</normaloff>:/icons/icons/explorer.ico</iconset>
   </property>
   <property name="text">
    <string>Eksplorator plików</string>
   </property>
  </action>
  <action name="actionDrukuj">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/print.ico</normaloff>:/icons/icons/print.ico</iconset>
   </property>
   <property name="text">
    <string>Drukuj</string>
   </property>
  </action>
  <action name="actionZa_aduj">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/open.ico</normaloff>:/icons/icons/open.ico</iconset>
   </property>
   <property name="text">
    <string>Załaduj</string>
   </property>
  </action>
  <action name="actionPoka_dane">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/eye.svg</normaloff>:/icons/icons/eye.svg</iconset>
   </property>
   <property name="text">
    <string>Pokaż dane</string>
   </property>
  </action>
  <action name="actionWyczy">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/eraser.svg</normaloff>:/icons/icons/eraser.svg</iconset>
   </property>
   <property name="text">
    <string>Wyczyść</string>
   </property>
  </action>
  <action name="actionWykres_s_upkowy_2">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/bar-chart.svg</normaloff>:/icons/icons/bar-chart.svg</iconset>
   </property>
   <property name="text">
    <string>Wykres słupkowy</string>
   </property>
  </action>
  <action name="actionWykres_liniowy">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/charts.svg</normaloff>:/icons/icons/charts.svg</iconset>
   </property>
   <property name="text">
    <string>Wykres liniowy</string>
   </property>
  </action>
  <action name="actionWykres_ko_owy">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/pie-chart.svg</normaloff>:/icons/icons/pie-chart.svg</iconset>
   </property>
   <property name="text">
    <string>Wykres kołowy</string>
   </property>
  </action>
  <action name="actionDodaj_opis">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/edit.ico</normaloff>:/icons/icons/edit.ico</iconset>
   </property>
   <property name="text">
    <string>Opis wykresu</string>
   </property>
  </action>
  <action name="actionZg_o_problem">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/help.ico</normaloff>:/icons/icons/help.ico</iconset>
   </property>
   <property name="text">
    <string>Zgłoś problem</string>
   </property>
  </action>
  <action name="actionQt">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/qt.ico</normaloff>:/icons/icons/qt.ico</iconset>
   </property>
   <property name="text">
    <string>Qt</string>
   </property>
  </action>
  <action name="actionGitHub">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/github.ico</normaloff>:/icons/icons/github.ico</iconset>
   </property>
   <property name="text">
    <string>GitHub</string>
   </property>
  </action>
  <action name="actionO_programie">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/info.ico</normaloff>:/icons/icons/info.ico</iconset>
   </property>
   <property name="text">
    <string>O programie</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
